cypress/integration/common/profile.js

```diff
@@ -28,9 +46,17 @@ When("I visit another user's profile page", () => {
   cy.openPage("profile/peter-pan");
 });
 
-Then("I cannot upload a picture", () => {
+Then("I cannot upload a profile picture", () => {
   cy.get(".base-card")
     .children()
     .should("not.have.id", "customdropzone")
     .should("have.class", "user-avatar");
 });
+
+
+Then("I cannot upload a profile header image", () => {
+  cy.get(".base-card")
+    .children()
+    .should("not.have.id", "profileHeaderDropzone")
+    .should("have.class", "profile-header");
```

Comment:
> [<img alt="DriesCruyskens" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/DriesCruyskens) **Authored by [DriesCruyskens](https://github.com/DriesCruyskens)**
_<time datetime="2020-06-26T16:38:08Z" title="Friday, June 26th 2020, 6:38:08 pm +02:00">Jun 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -28,9 +46,17 @@ When("I visit another user's profile page", () => {
   cy.openPage("profile/peter-pan");
 });
 
-Then("I cannot upload a picture", () => {
+Then("I cannot upload a profile picture", () => {
   cy.get(".base-card")
     .children()
     .should("not.have.id", "customdropzone")
     .should("have.class", "user-avatar");
 });
+
+
+Then("I cannot upload a profile header image", () => {
+  cy.get(".base-card")
+    .children()
+    .should("not.have.id", "profileHeaderDropzone")
+    .should("have.class", "profile-header-card");
```


I suspect the fix you did changing `profile-header` to `profile-header-card` was wrong after all and the errors you thought it caused had in fact different causes. Since this test finds the element with `.base-card` class and then analyzes its children, its easy to see why it can't find `profile-header-card`. The test in its original state was in fact checking the child with the `profile-header` class:

![image](https://user-images.githubusercontent.com/26900172/85880152-d48db900-b7db-11ea-822e-a48ce299eb4c.png)


tldr; The error regarding this test should be fixed by changing line 61 back to 
```suggestion
    .should("have.class", "profile-header");
```